bootstrap/handlers/headers/key_test.go

```diff
@@ -0,0 +1,97 @@
+//
+// Copyright (C) 2024 IOTech Ltd
```

Comment:
```suggestion
// Copyright (C) 2025 IOTech Ltd
```
update all the copyright to 2025

Reply:
Done